src/events-multiple/web/index.scss

```diff
@@ -1,5 +1,22 @@
 @import '_core';
 @import '_adverts';
+@import '_adverts-books';
```

Comment:
Do we need these css's to allow different styles? I assume yes if this is no longer masterclass specific but double checking

Reply:
I was going to remove them because we're only currently supporting masterclass and live events, but I think that the impact to the advert is minimal and it should make updating in the future much easier.
Happy to remove if we decide otherwise.